src/index.html

```diff
@@ -6,7 +6,455 @@
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="body">
+    <header class="header">
+      <a class="header__logo" href="#">
+        <img src="images/logo.png" alt="Moyo">
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__items"> 
```

Comment:
it shouldn't be plural. apply all over the page. Cause it is `list item`
```suggestion
          <li class="nav__item"> 
```